src/models/metadata.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const config = require('../services/config')
+
+function getMetadata () {
+  return {
+    public_key: config.getIn(['keys', 'ed25519', 'public']),
+    urls: {
```

Comment:
In this PR, I only moved as much of the existing code as I could without doing any refactoring. I plan to do anything else in a followup.